lxd/instance_get.go

```diff
@@ -152,5 +157,19 @@ func instanceGet(d *Daemon, r *http.Request) response.Response {
 		return response.SmartError(err)
 	}
 
+	if withEntitlements {
+		if recursion == 0 {
```

Comment:
Isn't recursion `0` just a list of URLs?

Reply:
in this endpoint (`GET /1.0/instances/{name}`), recusion=0 return an `*api.Instance` (with recursion=1, `*api.InstanceFull` is returned)

Reply:
fixed